external/storm-jdbc/src/main/java/org/apache/storm/jdbc/common/HikariCPConnectionProvider.java

```diff
@@ -40,6 +44,10 @@ public synchronized void prepare() {
             Properties properties = new Properties();
             properties.putAll(configMap);
             HikariConfig config = new HikariConfig(properties);
+            if(properties.containsKey("dataSource.url"))
+                LOG.info("DataSource Url: " + properties.getProperty("dataSource.url"));
+            else if (config.getJdbcUrl() != null)
+                LOG.info("JDBC Url: " + config.getJdbcUrl());
```

Comment:
The else-if is there because with datasource.className specified (e.g. com.mysql.jdbc.jdbc2.optional.MysqlDataSource), you either specify datasource.url (which takes precedence) or specify the jdbc property.